iOS/Flows/Conversation/Input/SwipeInputPanGestureHandler.swift

```diff
@@ -27,9 +27,11 @@ class SwipeInputPanGestureHandler {
     /// The center point of the preview view when the pan started.
     private var initialPreviewCenter: CGPoint?
     /// How far the preview view can be dragged left or right.
-    private let maxXOffset: CGFloat = 40
+    private let maxXOffset: CGFloat = 20
     /// If true, the preview view is currently in the drop zone.
     private var isPreviewInDropZone = false
+    /// The distance the user needs to drag in order to send a message at the highest delivery priority
+    private let totalDragDistance: CGFloat = 160
```

Comment:
See comment. 

This makes the drag distance consistent across all screen sizes.